### ISSUE TITLE
Re-training model

### ISSUE THREAD

**Issue by Michael-Yee** [zalandoresearch/flair#583]
Is there a way to re-train a model with new data?  

My scenario is that I labelled a small dataset to train my initial model.
I want to use this model to label other documents.
Then, I was to used these newly labelled documents in re-training the initial model.

Can I add new labelled data into the train, dev and test datasets of the initial model and just continue the training process?

Really, I'm looking for a way to cut down training time.  In TensorFlow you can do something like this and it so much faster than start from the beginning.

**Comment by alanakbik** [comment]
Hello @Michael-Yee yes I think something like this is possible the way you described. You could write a loop that trains the model for a few epochs, then extend the train, dev and test sets, and continue training, and repeat. We'd be curious to hear about your results and how well it works!

**Comment by deepankar27** [comment]
@Michael-Yee Can you please help me with answering the below question? What was the size of your data & did you get your desired accuracy?
